### PR TITLE
More help in migrating v1 to v2

### PR DESCRIPTION
**Pull request by andypillip**

### CODE REVIEW

File: content/pages/3.guides/2.upgrading-from-v1/index.md [hunk 1]
```diff
@@ -62,7 +62,10 @@ You know all those things that start with an underscore? `_site_root`, `_layout`
 
 ### Removed and changed fieldtypes
 
-The former `templates` fieldtype is now named `template`, singular.
+- `templates` fieldtype is now named `template`.
+- `checkbox` fieldtype is obsolete. Use `type: checkboxes` and define `options` with `1: Checkbox Label` for backwards-compatibility
```
Comment (jackmcdade) [inline content/pages/3.guides/2.upgrading-from-v1/index.md]: You could also mention the `toggle` fieldtype.

File: content/pages/3.guides/2.upgrading-from-v1/index.md [hunk 2]
```diff
@@ -62,7 +62,10 @@ You know all those things that start with an underscore? `_site_root`, `_layout`
...
+- `partial` is the new fieldtype that mimics `include`, and `field_order` is therefore obsolete.
+- `instructions` can only have on single value, `above` and `below` are obsolete
```
Comment (jackmcdade) [inline content/pages/3.guides/2.upgrading-from-v1/index.md]: have `a`

File: content/pages/3.guides/2.upgrading-from-v1/index.md (hunk 1 above)
Comment (andypillip) [inline content/pages/3.guides/2.upgrading-from-v1/index.md]: I will do, as soon as I use and understand that fieldtype. Or you mean as a checkbox-alternative?

File: content/pages/3.guides/2.upgrading-from-v1/index.md [hunk 3]
```diff
@@ -36,12 +36,14 @@ Each of these things may require a little bit of work. We've estimated that the
...
 - Mount them to their corresponding pages by adding `mount: collection_name` to `index.md`
-- Move `fields.yaml` from the page's to the collection's folder, and name it `folder.yaml`
+- Move `fields.yaml` from the page's to the collection's folder, and name it `folder.yaml`.
+- Change the `type` key to `order`, this sets whether to sort by number or date.
 
 ### Renamed/removed variables
 
 - `{{ layout_content }}` is now `{{ template_content }}` because it's the actual contents of the template. Doesn't that feel better?
 - `{{ _xml_header }}` is no longer necessary to render XML headers. Instead, use the `.xml` extension on your URL.
+- `{{ homepage }}` is now `{{ site_url }}`
```
Comment (jasonvarga) [inline content/pages/3.guides/2.upgrading-from-v1/index.md]: You can remove this. I've added `{{ homepage }}` to 2.0.5.

**Comment by jackmcdade** [comment]
If you can resolve the merge conflicts i will be happy to merge this in.

**Pull request closed without merging**